src/components/colorbar/attributes.js

```diff
@@ -169,6 +169,7 @@ module.exports = {
     showticksuffix: axesAttrs.showticksuffix,
     exponentformat: axesAttrs.exponentformat,
     showexponent: axesAttrs.showexponent,
+    exponentbase: axesAttrs.exponentbase,
```

Comment:
this would be a weird use case, but we should still add an image test with a non-10 exponent based colorbar.